### PR TITLE
Show TPS in top right corner

### PR DESCRIPTION
**Pull request by yellowmia11**
Show the TPS result and the change of the TPS result above the DPS value in the top right corner.

tpsdisplay3

### CODE REVIEW

**Comment by jokal2** [comment]
Yay! Thanks for this, I know nothing about UI code so things I added are likely stapled on poorly or just missing from the UI. 

Tangential to this, TPS is also incorrectly missing parry/block. We need some sort of UI toggle to set parry to `.145` in order for TPS information to be accurate. I currently just have a local patch that hardcodes it.

File: GUI/GUIControl.cpp [hunk 1]
```diff
@@ -884,46 +892,17 @@ void GUIControl::update_displayed_raid_dps_value(const double new_dps_value) {
...
-
-void GUIControl::update_displayed_tps_value(const double new_tps_value) {
-    double previous = last_personal_sim_result_tps;
-    last_personal_sim_result_tps = new_tps_value;
-    double delta = ((last_personal_sim_result - previous) / previous);
-    QString change = delta > 0 ? "+" : "";
-    change += QString::number(((last_personal_sim_result_tps - previous) / previous) * 100, 'f', 1) + "%";
-
-    QString tps = QString::number(last_personal_sim_result_tps, 'f', 2);
-    qDebug() << "Total TPS: " << tps;
```
Comment (jokal2) [inline GUI/GUIControl.cpp:905]: We should keep this output for consistency. ou can move it right back next to the `Total DPS` print.

Comment (yellowmia11) [reply GUI/GUIControl.cpp:905]: there was a typo in the output. it now prints "Total DPS", "Raid DPS", and "Total TPS", but not "Raid TPS". Is there any use for Raid TPS? seems irrelevant to me

**Comment by veluria** [comment]
Layout-wise, this is totally broken for me as well:

image

Running on Linux.

**Comment by lanza** [comment]
So the offsets are inversed on Darwin and Linux for some reason.

```
Text {
             id: continuousTpsUpdateText
             anchors {
                 top: parent.top
                 topMargin: -10
             }
```

the -10 has to be 10 for me on Darwin.

**Comment by yellowmia11** [comment]
I can't believe this is expected behaviour in Qt. Does anyone have an idea on how to change this so it works on all systems?

**Pull request merged**